drivers/docker/driver_test.go

```diff
@@ -3143,3 +3143,68 @@ func TestDockerDriver_GroupAdd(t *testing.T) {
 
 	require.Exactly(t, cfg.GroupAdd, container.HostConfig.GroupAdd)
 }
+
+func TestDriver_createImage_validateContainerAdmin(t *testing.T) {
```

Comment:
We can move this to `drivers/docker/driver_windows_test.go` with the appropriate build flags so that the tests compile on Linux.